packages/api/client.ts

```diff
@@ -352,7 +352,12 @@ export const getServiceStops = async (dbClient: Kysely<Database>, input: Service
 
     const stops = await dbClient
         .selectFrom("services")
-        .innerJoin("service_journey_patterns", "service_journey_patterns.operatorServiceId", "services.id")
+        .innerJoin("vehicle_journeys", "vehicle_journeys.lineRef", "services.lineId")
+        .innerJoin(
+            "service_journey_patterns",
+            "service_journey_patterns.journeyPatternRef",
+            "vehicle_journeys.journeyPatternRef",
+        )
```

Comment:
Could we use below code which will allow us to remove the where in in line 419 on `service_journey_patterns.operatorServiceId`

`.innerjoin("service_journey_patterns", (join) => join.onRef("service_journey_patterns.journeyPatternRef","=","vehicle_journeys.journeyPatternRef").on("service_journey_patterns. operatorServiceId","=","services.id"))`

Reply:
as discussed

Reply:
done